test/features/install_dataindex.feature

```diff
@@ -17,3 +17,29 @@ Feature: Kogito Data Index
       }
     }
     """
+
+#####
+
+  @externalcomponent
+  @infinispan
+  Scenario: Install Kogito Data Index with persistence using external Infinispan
+    Given Infinispan Secret "external-infinispan-secret" is created with credentials:
```

Comment:
@radtriste When I think about it the Infinispan secret and Infinispan instance steps could be merged together. The Infinispan secret will hardly have any relevance outside of Infinispan instance.

The credentials could be moved to Infinispan instance configuration. The Infinispan configuration then won't reflect the explicit content of CR, but it will simplify usage of this step and make it more readable and understandable.

Reply:
sure they can be merged :)

Reply:
On the other side we would lose flexibility in case we would like to specify more users for Infinispan.

Reply:
But I guess it is not a common usecase

Reply:
IMO if we really need it, then we will refactor that one.